src/test/java/com/google/javascript/cl2dts/closure.depgraph

```diff
@@ -0,0 +1,148 @@
+[
+  [
+    "nonroots",
+    [
+      [
+        "javascript/closure/base.js",
+        [
+          [
+            "custom_annotations",
+            []
+          ],
+          [
+            "is_externs",
```

Comment:
I don't understand any of these flags and fields.

Could you cut it down to just the stuff that's relevant for our test? Same for the file names, I don't think any reader of this test case will understand what that annotations canvas thing is.